### PR TITLE
Issue21

### LINKED ISSUE #21: Modify Arduino code to read "Duration" pin
Modify existing Arduino code to read in times when a fifth pin is high.

### PR DESCRIPTION
**Pull request by mimaxwell**
The Arduino is now able to read in when a fifth pin is high. 
Closes #21.

### CODE REVIEW

**Comment by cabeese** [comment]
Looks good. I'm excited to see all the masking and shifting explained.

@mimaxwell For some reason, I thought that you could only read in 4 pins at a time, but it looks like you're actually able to read in 8 and you're putting the "duration" pin adjacent to the others. Is that true? And do I have the terminology correct? What's the set of pins called? (A pin is the single metal thing sticking up out of the board, right?)

File: Read_PIND/Read_PIND.ino [hunk 1]
```diff
@@ -1,27 +1,31 @@
...
+
+
+uint8_t prevpinval = PIND & bitMask;
 uint8_t pinval;
 uint8_t xorpins;
 extern volatile unsigned long timer0_overflow_count;
 unsigned long time_elapsed;
 
-// longs are 32 bits, so the array can only hold 218 values
+// longs are 32 bits, so the array can only hold 210 values
```
Comment (cabeese) [inline Read_PIND/Read_PIND.ino]: Is this based on the amount of on-board memory that the board has? If so, perhaps we should reduce this a bit in order to allow for the SPI code to have some space to play around in down the road.

Comment (mimaxwell) [reply Read_PIND/Read_PIND.ino]: Yes. It will have to reduced when SPI is added, not sure by how much.

Comment (cabeese) [reply Read_PIND/Read_PIND.ino]: Works for me

**Pull request merged**